vendor/github.com/ductone/protoc-gen-apigw/apigw/v1/http_bridge.go

```diff
@@ -49,10 +49,9 @@ func TimeoutForRequest(req *http.Request) (time.Duration, bool) {
 }
```

Comment:
this change looks like a vendoring issue?

Reply:
hmmm tried go mod tidy and go mod vendor, but that doesn't seem to fix this.

Reply:
Fixed in latest commit